src/styles/partials/components/_emphasized-image.scss

```diff
@@ -55,8 +55,11 @@
 }
 
 @media screen and (max-width: $small-breakpoint) {
-  p:first-of-type,
+  p:first-of-type {
```

Comment:
A couple things

- i would leave it as and override the margin for last of type.
- you are missing .dg_card_content this would apply to p in this case (i think this is integration, so you might just be able to pull the latest and merge)
- confirmed with @sdurphy margin-large should be used here

```scss
.dg_card-content {
    p:first-of-type,
    p:last-of-type {
      margin: $base-spacing-unit * 3;
      text-align: center;
    }

    p:last-of-type {
      margin-bottom: $margin-large;
    }
  }
```